02week/excercises.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+let cars = ['focus','st', 'fiesta', 'model s'];
```

Comment:
instructions asked to put ford in the first position of this array, and honda in the last position of the next array